code/modules/clothing/head/misc_special.dm

```diff
@@ -195,8 +195,13 @@
 		if(anime || Holiday == APRIL_FOOLS_DAY)
 			speech.message = nekospeech(speech.message)
 
+/obj/item/clothing/head/kitty/equipped(var/mob/user, var/slot, hand_index = 0)
```

Comment:
I think you need to check whether `slot == slot_head` or this will fire when putting the item in hand

Reply:
Ironically, when I used slot == slot_head. It would only fire when equipped a second time to the head slot. Idk what the deal is with that, but ill look into it later I guess